pages/about.html

```diff
@@ -31,13 +31,12 @@
         
         <div class="mobile-menu">
             <ul class="mobile-items">
-                <li><a href="./pages/about.html"  class="mitem">About</a></li>
-                <li><a href="#features"  class="mitem">Features</a></li>
-                <li ><a href="#partners"  class="mitem">Partners</a></li>
-                <li><a href="./pages/about.html#email"  class="mitem">Contact Us</a></li>
-                <li><a href="#speaker-container"  class="mitem">Clients</a></li>
+                <li><a href="../pages/about.html"   class="mitem">About</a></li>
+                <li><a href="../index.html#features"  class="mitem">Features</a></li>
+                <li ><a href="../index.html#partners" class="mitem">Partners</a></li>
+                <li><a href="../pages/about.html#email"  class="mitem">Contact Us</a></li>
+                <li><a href="../index.html#speaker-container"  class="mitem">Clients</a></li>
```

Comment:
- [ ] I think it would be hreat if you decrease the height for the nabar to be exactly the same as the original design kindly fix it 🔥🔥

- You design
![image](https://user-images.githubusercontent.com/58553711/181490005-ab670878-782c-4b0a-81de-667c8e876fa7.png)

- Origina; design
![image](https://user-images.githubusercontent.com/58553711/181490081-b9bc5f24-19b1-47a1-b8ab-224043436741.png)